.eslintrc.json

```diff
@@ -10,7 +10,8 @@
       "rules": {
         "@typescript-eslint/no-floating-promises": "off",
         "@typescript-eslint/typedef": "off",
-        "no-unused-vars": "off"
+        "no-unused-vars": "off",
+        "react/react-in-jsx-scope": "off"
```

Comment:
This is needed because of the update to ui-kit that also force the use of emotion/react now